README.md

```diff
@@ -1,2 +1,24 @@
 # HW01
-Edit the readme
+## 07/06/2020
+**Hi Everyone**
+
+My name is *Sandra McClure* and I am a second year PHD student in *the committe on Molecular Metabolism and Nutrition*. I work in the Mimee Lab [https://voices.uchicago.edu/mimeelab/] on engineering the gut microbiome. I went to undergrad at Michigan State University for chemistry and biology, where I also did research in analytical electrochemistry. 
+
+Here is a picture of me from a pre-COVID era:
+![Image of me](https://voices.uchicago.edu/mimeelab/files/2020/06/Sandra_cropped.jpg)
```

Comment:
In mine, I added an image from my desktop, but it's great that you incorporated a photo from a link to a website!